<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Juego</class>
 <widget class="QWidget" name="Juego">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="fondoJuego">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/_7ee18ae9-e235-413a-a720-fa59a96e065b.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="comenzarBoton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>460</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/playBoton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cronometro">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;  2:00&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="reloj">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/cronometro.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ajustesBoton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/pauseBoton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="menuPausa">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>241</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/PAUSAMENU.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="musicaON">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Images/botonON.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="sonidoON">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Images/botonON.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Images/backBoton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="exitMenuButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Images/stopBoton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="musicaOFF">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Images/botonOFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="sonidoOFF">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Images/botonOFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="musicaOnLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Images/musica.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sonidoOnLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Images/sonido.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="musicaOffLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Images/musicaoff.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sonidoOffLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/Images/sonidooff.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="aceptarBoton">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>520</y>
     <width>101</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/BotonA.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="denegarBoton">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>520</y>
     <width>101</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/BotonD.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="flecha">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>320</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/Flecha.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pasaporteBoton">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>410</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/Icono Pasaporte.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nivel1">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/Nivel1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nivel2">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/Nivel2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nivel3">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/Nivel3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nivel4">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/Nivel4.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nivel5">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/Nivel5.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bienHecho">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>301</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/bienHecho.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="fallaste">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>311</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/fallaste.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="puntos">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>PUNTOS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reglas">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>600</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/rules.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>fondoJuego</zorder>
  <zorder>comenzarBoton</zorder>
  <zorder>reloj</zorder>
  <zorder>cronometro</zorder>
  <zorder>ajustesBoton</zorder>
  <zorder>menuPausa</zorder>
  <zorder>aceptarBoton</zorder>
  <zorder>denegarBoton</zorder>
  <zorder>flecha</zorder>
  <zorder>pasaporteBoton</zorder>
  <zorder>nivel1</zorder>
  <zorder>nivel2</zorder>
  <zorder>nivel3</zorder>
  <zorder>nivel4</zorder>
  <zorder>nivel5</zorder>
  <zorder>bienHecho</zorder>
  <zorder>fallaste</zorder>
  <zorder>puntos</zorder>
  <zorder>reglas</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
